kube_hunter/modules/hunting/apiserver.py

```diff
@@ -56,16 +56,18 @@ def __init__(self, evidence):
 
 
 class ApiInfoDisclosure(Vulnerability, Event):
+    """Information Disclosure depending upon RBAC permissions and Kube-Cluster Setup"""
     def __init__(self, evidence, using_token, name):
+        category=InformationDisclosure
         if using_token:
-            name += " using service account token"
+            name += " using default service account token"
```

Comment:
This does not have to be the service account token. depends on how the user runs kube-hunter.